src/internal/connector/onedrive/collections.go

```diff
@@ -325,6 +328,33 @@ func (c *Collections) Get(
 		folderPaths[driveID] = map[string]string{}
 		maps.Copy(folderPaths[driveID], paths)
 
+		// Set all folders in previous backup but not in the current
+		// one with DoNotMerge
```

Comment:
We actually want to tell kopia to delete them. Deleting them is safe because we must not have seen them during our item enumeration because they were deleted in OneDrive (we lookup folders by ID in the map so renames/moves don't affect this)

This is to catch situation like
```text
1. create backup and store Corso metadata
2. user deletes folder (e.x. /root/foo)
3. delta token expires
4. backup launched
5. all existing files and folders discovered (/root/foo not found since it was deleted)
```

steps 2 and 3 above can be swapped, the only important thing is that they both happen before the next backup

if we didn't make a collection and mark `/root/foo` as deleted then kopia wrapper would add it to the backup because kopia wrappers's default right now is "if it's not explicitly mentioned, assume it still exists at the same location with the same items"

```suggestion
		// one with state deleted.
```